packages/ndla-ui/src/Gloss/Gloss.tsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from '@emotion/styled';
+import { useTranslation } from 'react-i18next';
+import { colors, spacing, misc, fonts } from '@ndla/core';
+import { AccordionRoot, AccordionItem, AccordionHeader, AccordionContent } from '@ndla/accordion';
+import { Transcription } from '../model/Transcriptions';
+import SpeechControl from '../AudioPlayer/SpeechControl';
+
+interface Example {
+  example: string;
+  language: string;
+  transcriptions: Transcription;
+}
+
+export interface Props {
+  title: {
+    title: string;
+    language: string;
+  };
+  glossData: {
+    gloss: string;
+    wordClass?: string;
+    originalLanguage: string;
+    transcriptions: Transcription;
+    examples?: Example[][];
+  };
+  audio: {
+    title: string;
+    src?: string;
+  };
+}
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-color: ${colors.background.lightBlue};
+  padding: ${spacing.nsmall} ${spacing.normal};
+  border: 1px solid ${colors.brand.lighter};
+  border-radius: ${misc.borderRadius};
+  margin-bottom: ${spacing.xsmall};
+  gap: ${spacing.nsmall};
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const GlossContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: ${spacing.nsmall};
+`;
+
+const GlossSpan = styled.span`
+  font-weight: ${fonts.weight.bold};
+`;
+
+const TypeSpan = styled.span`
+  font-style: italic;
+`;
+
+const StyledAccordionHeader = styled(AccordionHeader)`
+  font-family: ${fonts.sans};
+  ${fonts.sizes('16px', '24px')};
+  font-weight: ${fonts.weight.semibold};
+  background-color: ${colors.background.lightBlue};
+`;
+
+const StyledAccordionContent = styled(AccordionContent)`
+  padding: 0;
+`;
+
+const TranslatedText = styled.div`
+  border-bottom: 1px solid ${colors.brand.lighter};
+  padding: ${spacing.small} ${spacing.normal};
+  font-family: ${fonts.sans};
+  ${fonts.sizes('18px', '24px')};
+  :first-child {
+    color: ${colors.brand.dark};
+    font-weight: ${fonts.weight.bold};
+    background-color: ${colors.background.lightBlue};
+  }
+`;
+
+const Gloss = ({ title, glossData, audio }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Container>
+        <Wrapper>
+          <GlossContainer>
+            <GlossSpan>{glossData.gloss}</GlossSpan>
+            {glossData.transcriptions.traditional && <span>{glossData.transcriptions.traditional}</span>}
+            {glossData.transcriptions.pinyin && (
+              <span key={t(`gloss.Transcriptions.pinyin`)}>{glossData.transcriptions.pinyin}</span>
+            )}
+            {glossData.wordClass && <TypeSpan>{t(`wordClass.${glossData.wordClass}`)}</TypeSpan>}
```

Comment:
Du har ingen indikasjon på kva som er kva her. Leseren har ingen måte å vite kva som er traditional og kva som er pinyin. I tillegg bør vel uansett orda markeres med språk slik at skjermlesere ikkje leser dei opp på norsk. 
```suggestion
            <GlossSpan lang="{glossData.originalLanguage}">{glossData.gloss}</GlossSpan>
            {glossData.transcriptions.traditional && <span lang="{glossData.originalLanguage}">{glossData.transcriptions.traditional}</span>}
            {glossData.transcriptions.pinyin && (
              <span key={t(`gloss.Transcriptions.pinyin`)} lang="{glossData.originalLanguage}">{glossData.transcriptions.pinyin}</span>
            )}
            {glossData.wordClass && <TypeSpan>{t(`wordClass.${glossData.wordClass}`)}</TypeSpan>}
```